docs/docs/reference/builtin-functions/feel-built-in-functions-boolean.md

```diff
@@ -55,3 +55,26 @@ is defined(x)
 is defined(x.y)
 // false - if no variable "x" exists or it doesn't have a property "y"
 ```
+
+## get or else(value, default)
+
+Return the provided value parameter if not `null`, otherwise return the default parameter
+
+**Function signature**
+
+```js
+get or else(value: Any, default: Any): Val
```

Comment:
:x: We should change the return type of the function to `Any`. 

`Any` is part of the DMN specification. `Val` is an implementation class of the engine. 

```suggestion
get or else(value: Any, default: Any): Any
```